sdk/communication/Azure.Communication.PhoneNumbers/tests/Infrastructure/PhoneNumbersClientLiveTestBase.cs

```diff
@@ -87,6 +87,12 @@ protected string GetTestPhoneNumber()
                 : TestEnvironment.CommunicationTestPhoneNumber;
         }
 
+        protected string GetUnathorizedNumber()
+        {
+            const string PhoneNumber = "+14255550123";
```

Comment:
Why do you need to define a getter here? 
Simply defining a variable should be good enough `public string PhoneNumber = "+14255550123";`